src/smk/tool/directions/panel-directions-options.html

```diff
@@ -8,6 +8,8 @@
     <template slot="commands">
     </template>
 
+    <enter-input v-bind:value="rangeKm" v-on:change="!busy && $$emit('change', {rangeKm: $event});">Range (km)</enter-input>
```

Comment:
We had talked about a slider before, but entering a numerical value is quick and easy to be precise. Though slider would maybe be preferable on mobile.